images/multus-cni/README.md

```diff
@@ -0,0 +1,28 @@
+<!--monopod:start-->
```

Comment:
How is the CI passing if the wolfi package is still in draft? 
 - https://github.com/wolfi-dev/os/pull/15476